### ISSUE TITLE
Hand-leading gestures

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7312]
**Preferred term label:**
Hand-leading gestures

**Synonyms**
Hand-taking gestures

**Definition (free text, please give PubMed ID)**
Taking other people by the hand to places they wish them to move to or taking someone’s hand to objects they want them to manipulate. PMID: 25388062

**Parent term (use hpo.jax.org/app)**
Impaired use of nonverbal behaviors
HP:0000758

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
added as HP:4000075

**Comment by bcoleman96** [comment]
Changed to HP:4000076
